README.md

```diff
@@ -327,6 +327,49 @@ interface. In case of a missing `interface_name`, the `name` of the profile name
 **Note:** The `name` (the profile name) and the `interface_name` (the device name) may be
 different or the profile may not be tied to an interface at all.
 
+### `match`
+
+Settings to specify with devices or systems match a profile. Currently, only the `path`
```

Comment:
This sentence does not read well to me. Maybe `s/with/which/`?

Reply:
Thanks. Updated, how about `Settings to specify devices or systems matching a profile. ` ?